README.md

```diff
@@ -64,58 +69,55 @@ it can be used independently:
-effectively lockout the account, the root user is not permitted to be specified
-as the target user account. The target user can be restricted to only login via
-ssh from specified hosts and also limited to only one login session at a time 
-for the execution of tasks. Multiple ssh keys can be specified for the target
-user to permit different user accounts on the controller to run bolt and provide
-a degree of attestation.
+The `simp_bolt` module can create a local user account on target systems,
+`simp_bolt` by default, that has the ability to `su` to the `root` user on the
+system. Every effort has been taken to implement this as securely as possible
+by including options to manage user security settings.
 
+Due to the potential to lock out the account, the `root` user is not permitted
+to be specified as the target user account. The target user can be restricted
+to only login via `ssh` from specified hosts and also limited to only one login
+session at a time for the execution of tasks. Multiple ssh keys can be
+specified for the target user to permit different user accounts on the
+controller to run `bolt` and provide a degree of attestation.
 
-The target user's home directory defaults to /var/local/simp_bolt. This location
-is used for temporary files on the target systems. This can be configured to a
-different location if desired.
+The target user's home directory defaults to `/var/local/simp_bolt`. This
+location is used for temporary files on the target systems. This can be
+configured to a different location if desired.
 
-Bolt logs are written to the /var/log/puppetlabs/bolt by default, and the 
+Bolt logs are written to `/var/log/puppetlabs/bolt` by default, and the
 directory structure will be created if necessary. This can also be configured
 to an alternate location.
 
-By default, Bolt collects various analytics associated with a random UUID, 
-non-identifiable user, details are available at 
+By default, Bolt collects various analytics associated with a random UUID,
+non-identifiable user, details are available at
 [Analytics data collection](https://puppet.com/docs/bolt/latest/bolt_installing.html#concept-8242)
-. The simp_bolt module overrides and disables this by default, but it can 
-be re-enabled in Hiera.
+. The `simp_bolt` module overrides and disables this by default, but it can be
+re-enabled in Hiera.
 
-The simp_bolt module leverages the `simp/pam` and `simp/sudo` modules 
-for implementation and will install them if necessary.
+The `simp_bolt` module optionally leverages the `simp/pam` and `simp/sudo`
+modules for implementation and will prompt for their installation if you
+attempt to use items that require them.
 
 ### Beginning with simp_bolt
 
-To configure a system as a Bolt controller, include the SIMP Bolt class and specify
-the system as a `bolt_controller` in Hiera.
+To configure a system as a Bolt controller, `include simp_bolt` and specify the
+system as a `bolt_controller` in Hiera.
 
 ```yaml
```

Comment:
In other PRs you had comments about `classes` applying to a SIMP system.  Do you need that here or is it implied because this is a SIMP profile?